app/models/abilities/organization_member_ability.rb

```diff
@@ -13,6 +13,7 @@ def initialize(permitting_organization_member)
       can :manage, OrganizationMember, organization_id: organization_id
       can :manage, ServiceSpecification, organization_id: organization_id
       can :manage, Payment, organization_id: organization_id
+      can :crud, Organization, id: organization_id
```

Comment:
Isn't crud to much? They shouldn't be able to create a organization, or am I wrong?

Reply:
tru